src/main/java/snapshotpolicy/sdk/sample/main.java

```diff
@@ -251,11 +247,11 @@ public static void main( String[] args )
 
         try
         {
-            await(Update.updateSnapshotPolicy(anfClient, resourceGroupName, anfAccountName, snapshotPolicyName, snapshotPolicyPatch));
+            Update.updateSnapshotPolicy(manager.serviceClient(), resourceGroupName, anfAccountName, snapshotPolicyName, snapshotPolicyPatch);
         }
-        catch (CloudException e)
+        catch (AzureException e)
```

Comment:
Any particular reasons you replaced the CloudException with the Azure Exception ?